elements/lisk-p2p/src/peer/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+export * from './inbound';
+export * from './base';
```

Comment:
we don't need to expose `base`?

Reply:
We do need because another files like p2p or peer_pool use methods from the base. Anyways we should rearrange these kind of things in short https://github.com/LiskHQ/lisk-sdk/issues/3678

Reply:
mmm ok let's clean up in the #3678